lib/src/table/macos_table_view.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:macos_ui/src/table/macos_table_order.dart';
+import './macos_table_header.dart';
+import './macos_table_row.dart';
+import './macos_table_datasource.dart';
+
+/// A scrollable data table with sorting and selection.
```

Comment:
Please add some more detail to the documentation here. Use the Human Interface Guidelines if you need ideas.

Additionally, please turn this doc into a template and use the macro on the constructor.